### ISSUE TITLE
mtx-grid columns width bug

### ISSUE THREAD

**Issue by ngiovin** [ng-matero/extensions#39]
![image](https://user-images.githubusercontent.com/59225969/95980208-1a1ed200-0e1d-11eb-9af1-61df66553c7b.png)
Hi, in last days I noticed a bug in my project using `mtx-grid`. It's like columns' width calculation fails.
Moreover the same bug is happening in the live demo at
https://ng-matero.github.io/extensions/components/data-grid/overview
I attach a snap from the demo.

My '@ng-matero/extensions' version is '9.9.0' and I had the same version when the app was working fine.

It looks like a problem with an involved dependency version, but it's only my assumption.

Any idea?

Tks

**Comment by nzbin** [comment]
I have found this issue caused by Chrome updating with version `86.0.4240.75`.

The easiest solution as following:

```css
.mtx-grid .mat-table:not(.mat-column-resize-table) {
    width: 100%;
}
```

**Comment by abilioconstantino** [comment]
Estou com o mesmo problema.... Só acontece com o chrome mesmo..

**Comment by ngiovin** [comment]
@nzbin Is your css fix not working?

**Comment by nzbin** [comment]
I will use JS to fix it.

**Comment by nzbin** [comment]
The issue has been fixed in `v9.11.7` and `v10.5.2`
